docs/.vitepress/config.mts

```diff
@@ -131,11 +127,9 @@ export default defineConfig({
         link: 'https://discord.gg/Tapeh8agYy'
       },
     ],
+
     footer: {
-      license: {
-        text: 'MIT License',
-        link: 'https://opensource.org/licenses/MIT'
-      },
+      message: '<a href="https://opensource.org/licenses/MIT" target="_blank">MIT License</a>',
```

Comment:
What is this license related to ? The license for kitops is Apache 2.0

Reply:
i can update it but this is not really used. We have a custom Vue component for the footer (only used in the homepage), but if we don't add this entry to the config, it wont render it _at all_. Its a Vitepress thing sadly...But ill use apache 2 just in case.

Reply:
nvm, i just updated the text to not mention any license